sw/device/silicon_creator/rom/BUILD

```diff
@@ -153,6 +153,17 @@ opentitan_rom_binary(
     ],
 )
 
+opentitan_rom_binary(
+    name = "rom_with_fake_keys",
+    testonly = True,
+    deps = [
+        ":linker_script",
+        ":rom_common",
+        ":rom_start",
+        "//sw/device/silicon_creator/rom/keys/fake",
+    ],
+)
+
```

Comment:
Checking that I understand this change correctly: would the final ROM binary be another target that looks the same as this one except is not testonly and depends on `//sw/device/silicon_creator/rom/keys/real`?

Reply:
Correct, that's the plan. Since we haven't generated any _real_ keys, I didn't define that target yet.